src/plugins/explore/public/components/visualizations/bar/bar_vis_options.tsx

```diff
@@ -15,8 +15,9 @@ import { TooltipOptionsPanel } from '../style_panel/tooltip/tooltip';
 import { AxesSelectPanel } from '../style_panel/axes/axes_selector';
 import { AllAxesOptions } from '../style_panel/axes/standard_axes_options';
 import { TitleOptionsPanel } from '../style_panel/title/title';
-import { AxisRole, VisFieldType } from '../types';
+import { AxisRole, VisFieldType, Threshold } from '../types';
```

Comment:
`Threshold` seems not used
```suggestion
import { AxisRole, VisFieldType } from '../types';
```